contents/handbook/people/team-structure/app-small-teams.md

```diff
@@ -0,0 +1,41 @@
+---
+title: App Small Teams
+sidebar: Handbook
+showTitle: true
+hideAnchor: false
+---
+
+App Small Teams are particular types of Small Teams that own an area of the product.
```

Comment:
Is "App" too overloaded a term

I've seen these called "product teams" or ["stream-aligned teams"](https://teamtopologies.com/key-concepts)

Reply:
I was choosing between "App team", "Product team" and "Feature team". On reflection I also think "Product team" makes the most sense and makes it clear the team is responsible for their product area holistically